CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.19
+
+### Fixes
+
+* instructLab now uses XDG based directories on macOS, similar to Linux.
```

Comment:
We've intentionally avoided using a `Fixes` section to keep the CHANGELOG focused solely on major changes. This will included in the list of fix commits but not a notable change. Otherwise, could you use the Feature section? Even though it's technically not a feature.

Also, this is somehow a breaking change existing users. We need a note to remove the previous config and cached models too.

Reply:
updated to breaking changes section and added the note
thanks!